gp-includes/things/translation.php

```diff
@@ -81,6 +81,11 @@ public function fix_translation( $translation ) {
 
 	public function restrict_fields( $translation ) {
 		$translation->translation_0_should_not_be( 'empty_string' );
+		$translation->translation_1_should_not_be( 'empty_string' );
+		$translation->translation_2_should_not_be( 'empty_string' );
+		$translation->translation_3_should_not_be( 'empty_string' );
+		$translation->translation_4_should_not_be( 'empty_string' );
+		$translation->translation_5_should_not_be( 'empty_string' );
```

Comment:
Yep, it works in my testing.

The change in `GP_Translation::for_translation()` makes sure that existing plurals with empty strings show up in 'Untranslated' view.

The new rules in `GP_Translation::restrict_fields()` make sure that new empty plurals cannot be submitted.